distributed/core.py

```diff
@@ -109,6 +109,106 @@ def _expects_comm(func: Callable) -> bool:
     return False
 
 
+class TaskGroup:
+    """Collection tracking all currently running asynchronous tasks within a group"""
+
+    #: If True, the group is closed and does not allow adding new tasks.
+    closed: bool
+
+    def __init__(self):
```

Comment:
if a constructor has no parameters, then the return type is required, otherwise it's considered untyped

```suggestion
    def __init__(self) -> None:
```